cloudinary.js

```diff
@@ -0,0 +1,28 @@
+//setting up our cloudinary account linking with multer upload files
+const cloudinary = require('cloudinary');
+
+const dotenv = require('dotenv');
+dotenv.config();
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
+});
+
+exports.uploads = (file, folder) => {
+  return new Promise((resolve) => {
```

Comment:
@iamikenna 

We are not supposed to use promises in our project. We have to replace it with `async-await`. 

Confirm this with someone if required.